src/V3EmitCSyms.cpp

```diff
@@ -156,6 +156,16 @@ class EmitCSyms final : EmitCBaseVisitor {
         string out = scpname;
         // Remove hierarchy
         string::size_type pos = out.rfind('.');
+
+        // if the last identifier is escaped,
+        // move to that esc in case there's a '.' in it
+        string::size_type esc = scpname.rfind('\\');
+        if (esc != string::npos) {
+            string escsub = scpname.substr(esc);
+            string::size_type endesc = escsub.find(' ');
+            if (endesc == escsub.length() - 1 || endesc == string::npos) { pos = esc - 1; }
```

Comment:
```suggestion
        // If the last identifier is escaped, move to that esc in case there's a '.' in it
        std::string::size_type esc = scpname.rfind('\\');
        if (esc != string::npos) {
            string escsub = scpname.substr(esc);
            std::string::size_type endesc = escsub.find(' ');
            if (endesc == escsub.length() - 1 || endesc == string::npos) pos = esc - 1;
```

Reply:
Wasn't this fixed with the automatic "Apply 'make format'" commit? And I'm passing the whitespace test case. Im pretty sure I have write access on:
<img width="269" alt="Screenshot 2023-01-10 at 10 34 17 AM" src="https://user-images.githubusercontent.com/24322714/211594041-b22540ef-e41b-4c30-9728-281687dc89c8.png">

Reply:
Think is ok now